repose-aggregator/core/repose-core/src/test/scala/org/openrepose/powerfilter/ReposeFilterChainTest.scala

```diff
@@ -36,44 +35,50 @@ import org.mockito.Matchers.{any, anyLong, same, eq => meq}
 import org.mockito.Mockito.{verify, when}
 import org.openrepose.commons.utils.http.PowerApiHeader.TRACE_REQUEST
 import org.openrepose.commons.utils.servlet.http.HttpServletRequestWrapper
-import org.openrepose.powerfilter.ReposeFilterChain.FilterContext
+import org.openrepose.core.systemmodel.config.{Filter => FilterConfig}
+import org.openrepose.powerfilter.ReposeFilterLoader.FilterContext
+import org.scalatest.LoneElement._
 import org.scalatest.junit.JUnitRunner
 import org.scalatest.mock.MockitoSugar
 import org.scalatest.{BeforeAndAfterEach, FunSpec, Matchers}
-import org.scalatest.LoneElement._
 import org.slf4j.MDC
+import org.springframework.context.support.AbstractApplicationContext
 import org.springframework.mock.web.{MockHttpServletRequest, MockHttpServletResponse}
 
 import scala.collection.JavaConverters._
 
 @RunWith(classOf[JUnitRunner])
 class ReposeFilterChainTest extends FunSpec with Matchers with MockitoSugar with BeforeAndAfterEach {
   var mockFilter: Filter = _
-  var mockRequest: MockHttpServletRequest = _
-  var mockResponse: MockHttpServletResponse = _
+  var mockAbstractApplicationContext: AbstractApplicationContext = _
+  var filterConfig: FilterConfig = _
   var originalChain: FilterChain = _
-  var metricsRegistry: MetricRegistry = _
   var timer: Timer = _
+  var metricsRegistry: MetricRegistry = _
   var tracer: MockTracer = _
+  var mockRequest: MockHttpServletRequest = _
+  var mockResponse: MockHttpServletResponse = _
 
   override protected def beforeEach(): Unit = {
     mockFilter = mock[Filter]
-    mockRequest = new MockHttpServletRequest("GET", "http://foo.com/bar")
-    mockResponse = new MockHttpServletResponse
+    mockAbstractApplicationContext = mock[AbstractApplicationContext]
```

Comment:
pretty sure you could have safely used null everywhere you use this

Reply:
True.
Updated.